modules/test/playwright/tests/journal-web/pages/JournalPage.ts

```diff
@@ -51,13 +51,11 @@ export class JournalPage {
 	}
 
 	async createBasicArticle(webContentName: string, text: string) {
-		await this.webContentTitleBox.fill(webContentName);
+		await this.webContentTitleBox.pressSequentially(webContentName, {
+			delay: 100,
+		});
```

Comment:
Since this is owned by another team, let's try to avoid making changes to this so we don't accidentally break their tests.